java/arcs/core/allocator/CollectionHandlePartitionMap.kt

```diff
@@ -94,7 +94,7 @@ class CollectionHandlePartitionMap(
 
```

Comment:
Question for the reviewer: Lines 74 - 88 seem like they can be refactored to use `storeAll` instead of multiple calls to `store`. Is there an advantage to one way vs the other?

Reply:
`storeAll` would probably work great, nice find!